python/README.md

```diff
@@ -31,7 +31,7 @@ Ambassador uses several TCP ports while running. All but one of them are in the
 | 8002 | `entrypoint/watcher`                  | Internal `watt` snapshot access; not exposed outside pod                                                           |
 | 8003 | `entrypoint/ambex`                    | Internal `ambex` snapshot access; not exposed outside pod                                                          |
 | 8004 | `diagd`                               | Internal `diagd` access when `AMBASSADOR_FAST_RECONFIGURE` is set; not exposed outside pod                         |
-| 8005 | `entrypoint/external_snapshot_server` | Exposes configuration snapshots for integration with other tools, such as the Ambassador Agent                     |
+| 8005 | `entrypoint/external_snapshot_server` | Exposes configuration snapshots for integration with other tools                                                   |
```

Comment:
Is this still real? 🤔

Reply:
We looked into this -- yup, the external snapshot server is still a thing, we'll leave it in for now. Maybe we'll modify the diagnostics page to use it. 😂